tests/models/test_model.py

```diff
@@ -79,24 +78,6 @@ def test_api_attrs():
 #         _MODELS["unknown"]
 
 
-def test_check_data(demda):
```

Comment:
Was this test transferred somewhere else prior to this PR?

Reply:
As far as I understand it is no longer necessary as this does a lot of type checking and name checking that is now done by the component and therefore no longer necessary. The dataset component tests include some equality checking which should also ensure the names are correct (at least I would assume). So as far as I understand we've covered all the necessary functionality in other tests.

Reply:
Ah yes I did not realize you actually removed the corresponding function so this is fine!